src/views/LandingView.vue

```diff
@@ -0,0 +1,124 @@
+<script lang="ts" setup>
+import Button from 'primevue/button'
+import InputText from 'primevue/inputtext'
+import { useRouter } from 'vue-router'
+
+const router = useRouter()
+const goToLoginPage = () => {
+  router.push('/login')
+}
+const goToRegisterPage = () => {
+  router.push('/sign-up')
+}
+</script>
+
+<template>
+  <nav class="navbar">
+    <div class="navbar-center">
+      <InputText placeholder="Search..." class="search-bar" />
+    </div>
+    <div class="sign-in-button">
+      <Button label="Sign In" @click="goToLoginPage()" />
+    </div>
+  </nav>
+  <div class="container wrapper">
```

Comment:
po co dwie klasy?
container to bardzo generyczna klasa.
Zobacz sobie BEM ;)

Reply:
jasne, dzięki:)